spec/index.html

```diff
@@ -421,6 +569,91 @@ <h2>Fragment identifiers</h2>
       <h3>Examples</h3>
       <p class="ednote">FIXME</p>
     </section>
+    <section id="faq" class="informative" data-format="markdown">
+      **NOTE** REMOVE THIS SECTION BEFORE PUBLICATION.
+
+      <h3>FAQ</h3>
+
+      #### Why YAML-LD does not preserve comments?
+      <p class="ednote">
+      [[JSON]] does not support comments.
+      While YAML-LD could define a specific predicate for that,
+      that is not enough because for example
+      in JSON the order of keywords is not preserved would
+      displace the comments.
```

Comment:
```suggestion
      While YAML-LD could define a specific predicate for comments,
      that is insufficient because, for example,
      the order of keywords is not preserved in JSON, so the
      comments could be displaced.
```